### PR TITLE
Amend UWSGI set up and Add Liveness Probe

### PR DESCRIPTION
**Pull request by Pheonnexx**
## What does this pull request do?

Amended UWSGI setup to increase http-workers to 5.

Added settings in UWSGI and created a new liveness probe that checks if a file is there or not.
 - Docker creates the file
 - WSGI alarm setting removes the file when it triggers
 - Liveness probe checks for the file, pod gets reset when that file is not found.

## Any other changes that would benefit highlighting?

Intentionally left blank.

## Checklist

- [ ] Provided JIRA ticket number in the title, e.g. "LGA-152: Sample title"

### CODE REVIEW

File: docker/cla_frontend.ini [hunk 1]
```diff
@@ -10,3 +10,5 @@ logformat={"process_name": "uwsgi", "timestamp_msec": %(tmsecs), "method": "%(me
 post-buffering = 1
 buffer-size=32768
 post-buffering-bufsize=32768
+http-workers=5
```
Comment (exonian) [inline docker/cla_frontend.ini:13]: Out of interest, what was this defaulting to previously?

**Review by exonian — APPROVED** [review]
Looks reasonable to me. As noted in the stack overflow answer we found this in, there is a risk of this terminating genuinely busy pods, but I think combined with the worker increase this is very worth it.

File: docker/cla_frontend.ini (hunk 1 above)
Comment (said-moj) [reply docker/cla_frontend.ini:13]: It was defaulting to 1 http worker, the number of http workers is displayed when uwsgi starts

File: Dockerfile [hunk 2]
```diff
@@ -50,6 +50,10 @@ COPY . .
 RUN chown -R app:app /home/app && \
     mkdir -p cla_frontend/assets
 
+# Add file for liveness probe
+RUN touch /tmp/listen_queue_healthy && \
+    chown www-data:www-data /tmp/listen_queue_healthy
```
Comment (said-moj) [inline Dockerfile:55]: I think `www-data:www-data` needs to be changed to `app:app`

**Pull request merged**